storm-core/src/clj/org/apache/storm/daemon/executor.clj

```diff
@@ -58,7 +58,7 @@
   (.prepare grouping context (GlobalStreamId. component-id stream-id) target-tasks)
   (if (instance? LoadAwareCustomStreamGrouping grouping)
     (fn [task-id ^List values load]
-      (.chooseTasks grouping task-id values load))
+      (.chooseTasks ^LoadAwareCustomStreamGrouping grouping task-id values load))
```

Comment:
@abhishekagarwal87  i see several places in that file which are missing type hints. Might be safer to limit this PR to just this perf issue.